dashboard/app.py

```diff
@@ -60,12 +58,13 @@ def update(
     global par_manager
     global opt_manager
     global cal_manager
-    # load data
-    exp_data, sim_data = load_data(db)
     # load input and output variables
     input_variables, output_variables, simulation_calibration = load_variables(
         state.experiment
     )
+    # load data
+    db = load_database(state.experiment)
```

Comment:
@EZoni 
I wonder if doing this inside the `update` function will increase latency. Do you have a way to check the timing of this type of call before and after this change?

Reply:
But now I wonder, maybe all of this is not necessary. I mean, I needed it because `load_config_dict` requires an `experiment` input argument. However, now that each configuration file contains one and only one experiment, it should not be necessary to carry around that information through all function calls, no? I can try to see if I can open a PR to simplify this.